dev-support/create-release/release-build.sh

```diff
@@ -136,19 +136,21 @@ if [[ "$1" == "tag" ]]; then
 
   git config user.name "$GIT_NAME"
   git config user.email "$GIT_EMAIL"
+  git config user.signingkey "${GPG_KEY}"
 
   # Create release version
   maven_set_version "$RELEASE_VERSION"
+  find . -name pom.xml -exec git add {} \;
   git add RELEASENOTES.md CHANGES.md
 
-  git commit -a -m "Preparing ${PROJECT} release $RELEASE_TAG; tagging and updates to CHANGES.md and RELEASENOTES.md"
+  git commit -s -m "Preparing ${PROJECT} release $RELEASE_TAG; tagging and updates to CHANGES.md and RELEASENOTES.md"
   log "Creating tag $RELEASE_TAG at the head of $GIT_BRANCH"
-  git tag "$RELEASE_TAG"
+  git tag -s -m "Via create-release" "$RELEASE_TAG"
```

Comment:
Maybe better if the message was meaningful to a user looking at the signed tag? Back in 1.1, I used to use "Tag $VERSION $RC", i.e., `git tag -s 1.1.2RC1 -m "Tag 1.1.2RC1"`.

These the only places we commit manually?

Why do we not use the maven release plugin to inc the version number and handle the commits for us? (separate issue)

Reply:
Your suggestion for the comments (on second thoughts) add no value. We are just adding the tag name as comment; i.e. adding no net info?

Let me combine your suggestion and the original message ; I think it important to note the tag made by the release scripts as opposed to by a human.

Tell me more about mvn release including the version number?

Reply:
In other projects, I used to use `release:prepare` to apply version change to the poms and create a release tag. A little configuration of the plugin lets you specify these details, it worked pretty well. I was never brave enough to let the plugin push changes though ; i always verified and pushed manually.

Reply:
https://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html

Reply:
Oh. That looks useful. HBASE-25466. Thanks.